pkgs/development/compilers/dotnet/build-dotnet.nix

```
@@ -127,4 +150,4 @@ stdenv.mkDerivation rec {
     mainProgram = "dotnet";
     platforms = builtins.attrNames srcs;
```

Not really related to this PR, but this would be preferable:
```suggestion
    platforms = attrNames srcs;
```
`attrNames` is already in scope because of `with lib`.